sources/T-FG-AI4H-2022-MSW-E.adoc

```diff
@@ -0,0 +1,7756 @@
+= Ethics and governance of artificial intelligence for health
+:bureau: T
+:docnumber: FG-AI4H DEL01
+:series: Focus Group on Artificial Intelligence for Health (FG-AI4H)
+:published-date: 2022-06-02
+:copyright-year: 2022
+:edition: 1
+:status: in-force
+:recommendationnumber: ITU-T FG-AI4H
+:imagesdir: images
+:doctype: focus-group
+:docfile: T-FG-AI4H-2022-MSW-E.adoc
```

Comment:
@anermina we no longer need the `docfile` attribute, and it is deprecated, please always remove it. thanks!